backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -7,9 +7,10 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-You back up applications by creating a `Backup` custom resource (CR). See xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc#oadp-creating-backup-cr-doc[Creating a Backup CR].
+You back up applications by creating one or more `Backup` custom resources (CR). See xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc#oadp-creating-backup-cr-doc[Creating a Backup CR].
 
-* The `Backup` CR creates backup files for Kubernetes resources and internal images on S3 object storage.
+* A `Backup` CR creates backup files for Kubernetes resources and internal images on S3 object storage.
+* You need to create one `Backup` CR per storage location. For example, to perform a backup that includes a backup of a local snapshot and a backup of an offsite location, you need to create two `Backup` CRs, one for each location.
```

Comment:
I don't think we need add this correlation of backup and backupstoragelocation. You can have multiple backup CRs using the same BSL or they can also use different BSLs